packages/salesforcedx-utils-vscode/src/helpers/utils.ts

```diff
@@ -29,12 +29,13 @@ export const extractJsonObject = (str: string): any => {
 //
 // To get around this, fs.realpathSync.native() is called to get the
 // URI with the actual file name.
+
 export const flushFilePath = (filePath: string): string => {
   if (filePath === '') {
     return filePath;
   }
 
-  let nativePath = realpathSync.native(filePath);
+  let nativePath = isSymbolicLink(filePath)? filePath: realpathSync.native(filePath);
```

Comment:
@mingxuanzhangsfdx it would be nice to not use `realpathSync.native` and use `realpathSync` instead. There are some limitations with `native` function on windows with the primary being max file path is limited to 260 characters, which could become an issue.